lib/l10n/app_it.arb

```diff
@@ -0,0 +1,600 @@
+  "renameEntrySetPagePatternFieldLabel": "Pattern per i nomi",
+  "renameEntrySetPageInsertTooltip": "Inserisci campo",
+  "renameEntrySetPagePreview": "Anteprima",
+
+  "renameProcessorCounter": "Contatore",
+  "renameProcessorName": "Nome",
+
+  "deleteSingleAlbumConfirmationDialogMessage": "{count, plural, =1{Cancellare questo album e i suoi elementi?} other{Cancellare questo album e i suoi {count} elementi?}}",
+  "deleteMultiAlbumConfirmationDialogMessage": "{count, plural, =1{Cancellare questi album e i loro elementi?} other{Cancellare questi album e i loro {count} elementi?}",
+
+  "exportEntryDialogFormat": "Formato:",
+  "exportEntryDialogWidth": "Larghezza",
+  "exportEntryDialogHeight": "Altezza",
+
+  "renameEntryDialogLabel": "Nuovo nome",
+
+  "editEntryDateDialogTitle": "Data e ora",
+  "editEntryDateDialogSetCustom": "Imposta data personalizzata",
+  "editEntryDateDialogCopyField": "Copia da un'altra data",
+  "editEntryDateDialogCopyItem": "Copia da altro elemento",
+  "editEntryDateDialogExtractFromTitle": "Estrai dal titolo",
+  "editEntryDateDialogShift": "Turno",
+  "editEntryDateDialogSourceFileModifiedDate": "Data di modifica del file",
+  "editEntryDateDialogTargetFieldsHeader": "Campi da modificare",
+  "editEntryDateDialogHours": "Ore",
+  "editEntryDateDialogMinutes": "Minuti",
+
+  "editEntryLocationDialogTitle": "Posizione",
+  "editEntryLocationDialogChooseOnMapTooltip": "Scegli sulla mappa",
+  "editEntryLocationDialogLatitude": "Latitudine",
+  "editEntryLocationDialogLongitude": "Longitudine",
+
+  "locationPickerUseThisLocationButton": "Usa questa posizione",
+
+  "editEntryRatingDialogTitle": "Valutazione",
+
+  "removeEntryMetadataDialogTitle": "Rimozione dei metadati",
+  "removeEntryMetadataDialogMore": "Altro",
+
+  "removeEntryMetadataMotionPhotoXmpWarningDialogMessage": "XMP è richiesto per riprodurre il video all'interno di una foto in movimento.\n\nSei sicuro di volerlo rimuovere?",
+
+  "videoSpeedDialogLabel": "Velocità di riproduzione",
+
+  "videoStreamSelectionDialogVideo": "Video",
+  "videoStreamSelectionDialogAudio": "Audio",
+  "videoStreamSelectionDialogText": "Sottotitoli",
+  "videoStreamSelectionDialogOff": "Off",
+  "videoStreamSelectionDialogTrack": "Traccia",
+  "videoStreamSelectionDialogNoSelection": "Non ci sono altre tracce",
+
+  "genericSuccessFeedback": "Fatto!",
+  "genericFailureFeedback": "Fallito",
+
+  "menuActionConfigureView": "Vista",
+  "menuActionSelect": "Seleziona",
+  "menuActionSelectAll": "Seleziona tutto",
+  "menuActionSelectNone": "Seleziona nessuno",
+  "menuActionMap": "Mappa",
+  "menuActionStats": "Statistiche",
+
+  "viewDialogTabSort": "Ordina",
+  "viewDialogTabGroup": "Gruppo",
+  "viewDialogTabLayout": "Layout",
+
+  "tileLayoutGrid": "Griglia",
+  "tileLayoutList": "Lista",
+
+  "aboutPageTitle": "Informazioni",
+  "aboutLinkSources": "Fonti",
+  "aboutLinkLicense": "Licenza",
+  "aboutLinkPolicy": "Privacy Policy",
```

Comment:
Samsung / Google use `Informativa sulla privacy` / `Norme sulla privacy`. I'll use one of these. Any preference?

Reply:
Right, `Informativa sulla privacy` sounds better